supervisor/daemon/wireguard/wginterface/interface_windows.go

```diff
@@ -22,17 +22,25 @@ import (
 	"net"
 
 	"github.com/rs/zerolog/log"
-
+	"golang.zx2c4.com/wireguard/device"
 	"golang.zx2c4.com/wireguard/ipc"
 	"golang.zx2c4.com/wireguard/tun"
+
+	"github.com/mysteriumnetwork/node/utils/cmdutil"
 )
 
 func createTunnel(interfaceName string) (tunnel tun.Device, _ string, err error) {
 	log.Info().Msg("Creating Wintun interface")
-	wintun, err := tun.CreateTUN(interfaceName, 0)
+	wintun, err := tun.CreateTUN(interfaceName, device.DefaultMTU)
 	if err != nil {
 		return nil, interfaceName, fmt.Errorf("could not create Wintun tunnel: %w", err)
```

Comment:
We are passing `interfaceName` and then returing that `interfaceName`, wtf?

Reply:
If you look and darwin implementation of this, you will notice that the requested interface name and resulting name might be different, e.g.: `utun` -> `utun6`